easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb

```diff
@@ -0,0 +1,57 @@
+easyblock = 'EB_GCC'
+
+name = 'GCCcore'
+version = '6.1.0'
+
+homepage = 'http://gcc.gnu.org/'
+description = """The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Java, and Ada,
+ as well as libraries for these languages (libstdc++, libgcj,...)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+mpfr_version = '3.1.4'
+
+source_urls = [
+    'http://ftpmirror.gnu.org/gmp',  # idem for GMP
+    'http://ftpmirror.gnu.org/mpfr',  # idem for MPFR
+    'http://www.multiprecision.org/mpc/download',  # MPC official
+    'ftp://gcc.gnu.org/pub/gcc/infrastructure/',  # GCC dependencies
+    'http://gcc.cybermirror.org/infrastructure/',  # HTTP mirror for GCC dependencies
+    'http://ftp.heanet.ie/mirrors/gnu/%(namelower)s/%(namelower)s-%(version)s/', # Alternative for GCC
+    'http://ftpmirror.gnu.org/gcc/gcc-%(version)s',  # GCC auto-resolving HTTP mirror
+]
+
+sources = [
+    'gcc-%(version)s.tar.bz2',
+    'gmp-6.1.1.tar.bz2',
+    'mpfr-%s.tar.bz2' % mpfr_version,
+    'mpc-1.0.3.tar.gz',
+    'isl-0.16.1.tar.bz2',
+]
+
+builddependencies = [
+    ('M4', '1.4.17'),
+    ('binutils', '2.27'),
+]
+
+patches = [
+    ('%s-%s_mpfr-%s-allpatches-20160804.patch' % (name, version, mpfr_version), '../mpfr-%s' % mpfr_version),
```

Comment:
@AFCJamie no need for the `%(name)s-%(version)s_mpfr...`, just use the existing patch with `mpfr...`